airflow/providers/google/cloud/operators/dataplex.py

```diff
@@ -610,3 +612,733 @@ def execute(self, context: Context) -> None:
+                region=self.region,
+                fail_on_dq_failure=self.fail_on_dq_failure,
+                timeout=self.timeout,
+            )
+        return job_id
+
+
+class DataplexGetDataQualityScanResultOperator(GoogleCloudBaseOperator):
+    """
+    Gets a Data Scan Job resource.
+
+    :param project_id: Required. The ID of the Google Cloud project that the lake belongs to.
+    :param region: Required. The ID of the Google Cloud region that the lake belongs to.
+    :param data_scan_id: Required. Data Quality scan identifier.
+    :param job_id: Optional. Data Quality scan job identifier.
+    :param api_version: The version of the api that will be requested for example 'v1'.
+    :param retry: A retry object used  to retry requests. If `None` is specified, requests
+        will not be retried.
+    :param timeout: The amount of time, in seconds, to wait for the request to complete. Note that if
+        ``retry`` is specified, the timeout applies to each individual attempt.
+    :param metadata: Additional metadata that is provided to the method.
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :param fail_on_dq_failure: If set to true and not all Data Quality scan rules have been passed,
+        an exception is thrown. If set to false and not all Data Quality scan rules have been passed,
+        execution will finish with success.
+    :param wait_for_result: Flag indicating whether to wait for the result of a job execution
+        or to return the job in its current state.
+
+    :return: Dict representing DataScanJob.
+        When the job completes with a successful status, information about the Data Quality result
+        is available.
+    """
+
+    template_fields = ("project_id", "data_scan_id", "impersonation_chain")
+
+    def __init__(
+        self,
+        project_id: str,
+        region: str,
+        data_scan_id: str,
+        job_id: str | None = None,
+        api_version: str = "v1",
+        retry: Retry | _MethodDefault = DEFAULT,
+        timeout: float | None = None,
+        metadata: Sequence[tuple[str, str]] = (),
+        gcp_conn_id: str = "google_cloud_default",
+        impersonation_chain: str | Sequence[str] | None = None,
+        fail_on_dq_failure: bool = False,
+        wait_for_results: bool = True,
```

Comment:
```suggestion
        wait_for_results: bool = True,
        execution_timeout: float = 30,
```

Reply:
maybe ~5min would be a better option

Reply:
the time should be much longer, it was agreed that 10 min

Reply:
makes sense, thanks!